src/main/java/de/sakpaas/backend/service/SearchService.java

```diff
@@ -104,6 +105,14 @@ protected SearchRequest checkForBrands(SearchRequest request) {
    * @return the updated Request Object
    */
   protected SearchRequest getCoordinatesFromNominatim(SearchRequest request) {
+    CoordinateDetails coordinateDetails = new CoordinateDetails(0.0, 0.0);
```

Comment:
Solltest du hier nicht die eventuell vorhandenen Koordinaten aus dem SearchRequest nehmen?

Reply:
@RobSpot-Gaming  Schau dir die Zeilen danach an !

Reply:
myBad

Reply:
wäre es dann nicht sinnvoller die returns in dem try-catch-Block zu haben?